packages/components/src/Contributors/ContributorCount.tsx

```diff
@@ -7,15 +7,31 @@ export interface ContributorsCountProps {
   totalContributors: number;
 }
 
-export const ContributorCount: React.FunctionComponent<ContributorsCountProps> = props => {
-  return (
-    <ContributorCountStyled>
-      <ContributorCountAvatars>
-        {props.displayedContributors.map((contributor, idx) => (
-          <img src={contributor.avatar} key={idx} />
-        ))}
-      </ContributorCountAvatars>
-      <ContributorCountTotal>{props.totalContributors + " contributors"}</ContributorCountTotal>
-    </ContributorCountStyled>
-  );
-};
+export class ContributorCount extends React.Component<ContributorsCountProps> {
+  public render(): JSX.Element {
+    return (
+      <ContributorCountStyled>
+        {this.props.totalContributors !== 0 ? (
+          this.renderContributers()
+        ) : (
+          <ContributorCountTotal>Be the first to tip</ContributorCountTotal>
+        )}
+      </ContributorCountStyled>
+    );
+  }
+  private renderContributers = (): JSX.Element => {
+    return (
+      <>
+        <ContributorCountAvatars>
+          {this.props.displayedContributors.slice(0, 3).map((contributor: any, i: number) => {
+            return <img src={contributor.payerChannel.tiny72AvatarDataUrl} key={i} />;
```

Comment:
we'll need to show a placeholder icon here if channel doesn't have one set. should probably build out a component for this based on Rob's "default avatar" designs (solid color background, with first letter of handle)

Reply:
I thought we were auto assigning a default avatar if you don't choose one?